pkg/server/node.go

```diff
@@ -2979,3 +2980,30 @@ func (n *Node) GetRangeDescriptors(
 		RangeDescriptors: rangeDescriptors,
 	})
 }
+
+func (n *Node) TenantInfo(
```

Comment:
As discussed, I _kinda_ expected this to live on the admin server.

We should think about authorization here.  A tenant should only be able to request the info for the tenant it is authorized as.

Ahh, I see now that you have a TODO for this elsehwere.